src/rocksdb/src/config.rs

```diff
@@ -460,12 +563,24 @@ mod tests {
             defaults::DEFAULT_PARALLELISM,
             defaults::DEFAULT_COMPRESSION_TYPE,
             defaults::DEFAULT_BOTTOMMOST_COMPRESSION_TYPE,
+            defaults::DEFAULT_BATCH_SIZE,
         )
         .unwrap();
 
         assert_eq!(r, RocksDBTuningParameters::default());
     }
 
+    #[test]
```

Comment:
The linter will complain about this once https://github.com/MaterializeInc/materialize/pull/19603 lands, please rebase your change then and use `#[mz_ore::test]`.
```suggestion
    #[mz_ore::test]
```

Reply:
thanks for letting me know!